vault-k8s.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: vault-k8s
+  version: 1.2.0
+  epoch: 0
+  description: Tool for encryption as a service, secrets and privileged access management
+  copyright:
+    - license: MPL-2.0
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/hashicorp/vault-k8s
+      tag: v${{package.version}}
+      expected-commit: e4d24097d597b62b1d57cb808a4f984fc41419f0
+      destination: vault-k8s
+  - if: ${{build.arch}} == 'aarch64'
+    runs: |
+      cd vault-k8s
+      # Our builtin LDFLAGS conflict with some makefile defined Go-specific ones.
+      unset LDFLAGS
+      # The Makefile hardcodes amd64 :()
+      make build GOARCH=arm64
+  - if: ${{build.arch}} == 'x86_64'
+    runs: |
+      cd vault-k8s
+      # Our builtin LDFLAGS conflict with some makefile defined Go-specific ones.
+      unset LDFLAGS
+      make build GOARCH=amd64
+  - runs: |
+      mkdir -p ${{targets.destdir}}/usr/bin
+      install -m755 -D vault-k8s/dist/vault-k8s "${{targets.destdir}}/usr/bin/"
+  - uses: strip
+update:
```

Comment:
nice, when did this get added :)

Reply:
recently - there's some docs here https://github.com/wolfi-dev/os/blob/main/docs/UPDATES.md, we're rolling it out to cover a lot more packages where possible.